src/main/java/com/raisetech/task10/controller/NameController.java

```diff
@@ -1,55 +1,72 @@
-package com.raisetech.task10.controller;
-
-import com.raisetech.task10.form.CreateForm;
-import com.raisetech.task10.service.NameService;
-import com.raisetech.task10.entity.Student;
-import org.springframework.http.ResponseEntity;
-import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.*;
-import org.springframework.web.util.UriComponentsBuilder;
-
-import java.net.URI;
-import java.util.List;
-import java.util.Map;
-
-@RestController
-public class NameController {
-
-    private final NameService nameService;
-
-    public NameController(NameService nameService) {
-        this.nameService = nameService;
-    }
-
-    @GetMapping("/names")
-    public List<Student> getNames() {
-        return nameService.findAll();
-    }
-
-    @GetMapping("/names/{id}")
-    public Student getUser(@PathVariable("id") int id) throws Exception {
-        return nameService.findById(id);
-    }
-
-    @PostMapping("/names")
-    public ResponseEntity<Map<String, String>> createUser(@RequestBody @Validated CreateForm name, UriComponentsBuilder uriBuilder) {
-        nameService.createUser(name);
-        URI url = uriBuilder
-                .path("names/" + name.getId())
-                .build()
-                .toUri();
-        return ResponseEntity.created(url).body(Map.of("message", "登録が完了しました。"));
-    }
-
-    @PatchMapping("names/{id}")
-    public Map<String, String> updateUser(@PathVariable("id") int id, @RequestBody @Validated CreateForm name) {
-        nameService.updateUser(id, name);
-        return Map.of("message", "更新が完了しました。");
-    }
-
-    @DeleteMapping("names/{id}")
-    public Map<String, String> deleteUser(@PathVariable int id) throws Exception {
-        nameService.deleteById(id);
-        return Map.of("message", "削除しました。");
-    }
-}
+package com.raisetech.task10.controller;
+
+import com.raisetech.task10.form.CreateForm;
+import com.raisetech.task10.service.NameService;
+import com.raisetech.task10.entity.Student;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class NameController {

```

Comment:
学生を扱うならクラス名も変更すべきですね！
```suggestion
public class StudentController {

```

Reply:
ServiceやMapperも同じくです。